HerPublicWebsite.UnitTests/BusinessLogic/ExternalServices/EmailSending/GovUkNotifyApiTests.cs

```diff
@@ -134,4 +147,28 @@ public void SendPendingReferralReportEmail_WhenCalled_DoesNotCallSendEmailIfEmai
             It.IsAny<Dictionary<string, dynamic>>(),
             null, null), Times.Never);
     }
+    
+    [TestCase(1, 10, 2022, "01/10/2022")]
+    [TestCase(11, 1, 2023, "11/01/2023")]
+    [TestCase(29, 2, 2024, "29/02/2024")]
```

Comment:
Nitpick: not sure we benefit from having three cases here - is there something you're trying to cover here that I'm not appreciating?

Reply:
The first two were for testing single digit day and single digit month both format as intended. The third was leap day testing since its a date, but its probably not necessary.

Reply:
I've updated this test to be when both month and day are double digits